src/Data/Validation.hs

```diff
@@ -660,14 +675,12 @@ optional (Just a) f =
     DisputedCtx gfs lfs b -> DisputedCtx gfs lfs (Just b)
     RefutedCtx gfs lfs    -> RefutedCtx gfs lfs
 
--- | Runs a set of validations only when a requirement is met.
-validateWhen :: Bool -> VCtx f b -> VCtx f (Maybe b)
-validateWhen False _ = pure Nothing
-validateWhen True v  = fmap Just v
-
--- | Runs a set of validations when a requirement is not met.
-validateUnless :: Bool -> VCtx f b -> VCtx f (Maybe b)
-validateUnless b = validateWhen (not b)

```

Comment:
If you guys are going to outright axe some methods consider using the [DEPRECATED pragma](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/glasgow_exts.html#warning-and-deprecated-pragmas) instead of forcing yourself to do a major version bump.

Reply:
not axing, these were introduced earlier in this PR

Reply:
Ah ok, kindly ignore this then 😄

Reply:
These never made it into a released version.